dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -121,19 +121,44 @@ public final PreparedStatement createStatementWithBoundFixtureSymbols(
     }
 
     @Override
+    public final StatementExecution createStatementExecutionWithBoundFixtureSymbols(TestHost th, String commandText)
+            throws SQLException {
+        return createStatementExecution(createStatementWithBoundFixtureSymbols(th, commandText));
+    }
+
+    @Override
     public DdlStatementExecution createDdlStatementExecution(String ddl)
             throws SQLException {
         return new DdlStatementExecution(getConnection().createStatement(), ddl);
     }
 
+    /**
+     * Create a procedure statement execution object for the given prepared statement
+     */
+    protected StatementExecution createStatementExecution(PreparedStatement statement) {
+        return new StatementExecution(statement);
+    }
+
+    /**
+     * Create a function statement execution object for the given prepared statement
+     */
+    protected StatementExecution createFunctionStatementExecution(PreparedStatement statement) {
```

Comment:
Would it be desirable to have this method accepting the SQL statement text (like `createDdlStatementExecution`)?